tutorial3_distributions/demo/main.js

```diff
@@ -66,74 +73,94 @@ function init() {
   svg = d3
     .select("#d3-container")
     .append("svg")
-    .attr("width", constants.width)
-    .attr("height", constants.height);
+    .attr("width", width)
+    .attr("height", height);
 
   // add the xAxis
   svg
     .append("g")
-    .attr("class", "x-axis")
-    .attr(
-      "transform",
-      `translate(0,${constants.height - constants.margin.bottom})`
-    )
-    .call(xAxis);
-
-  // add the xAxis
+    .attr("class", "axis x-axis")
+    .attr("transform", `translate(0,${height - margin.bottom})`)
+    .call(xAxis)
+    .append("text")
+    .attr("class", "axis-label")
+    .attr("x", "50%")
+    .attr("dy", "3em")
+    .text("Ideology Rating");
```

Comment:
axes labels -- it adds some code but I think it is a good example to give them. That said, I am on the fence about including it so let me know what you think. 

My motivation was to try to give them all the tools necessary to be able to create a good project out of this -- but maybe this is too much spoon feeding?

Reply:
dope